scripts/src/main/resources/scripts/command/rewrite

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "version help"
+  fi
+  exit
+fi
+
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+function doRun() {
+    local recipe="${1}"
+    local tool="${2}"
+
+    doRunCommand "mvn -U org.openrewrite.maven:rewrite-maven-plugin:run \
+      -Drewrite.recipeArtifactCoordinates=${tool} \
```

Comment:
According to the help, the `tool` is optional.
Shouldn't we omit this entire line if `tool` is not present?

Can one of the arguments contain spaces? I assume not, otherwise you would need to put single-quotes around each arg-line.

Reply:
The arguments can not contain spaces since the syntax is always something like `groupId:artifactId:version`. 

We can omit the line with `tool` but there is no error when the second parameter with `tool` is empty.

Reply:
OK, fine. Thanks for clarification.